docs/UserGuide.md

```diff
@@ -36,87 +57,61 @@ Refer to the [Features](#features) below for details of each command.
 **Notes about the command format:** <br />
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+e.g. in `guest add n/NAME`, `NAME` is a parameter which can be used as `guest add n/John Doe`.
 
 * Items in square brackets are optional.<br>
 e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Items with `…`​ after them can be used multiple times including zero times.
+e.g. `[t/TAG...]` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
```

Comment:
let's standardise how we represent tags. i think the [t/TAG]... (with the closely-spaced ZWSP thing) will be clearer. what do you think?